lib/Cro/Uri.pm6

```diff
@@ -46,6 +45,12 @@ class Cro::Uri {
     #| Given the example "https://user@cro.services:44433/example/url?foo=bar&x=42#here",
     #| this would return "here"
     has Str $.fragment;
+}
+
+#| An immutable representation of a URI
+class Cro::Uri does Cro::ResourceIdentifier {
+    #| The kind of host found in the URI (a domain name or some kind of IP address)
+    enum Host <RegName IPv4 IPv6 IPvFuture>;
```

Comment:
Can this go away now we have `Cro::ResourceIdentifier::Host`?

Reply:
I'm unsure why this is left.